bigtable-hbase-1.x-parent/bigtable-hbase-1.x/src/test/java/com/google/cloud/bigtable/hbase1_x/TestBigtableConnection.java

```diff
@@ -94,11 +94,12 @@ public void tearDown() throws IOException {
 
   @Test
   public void testOverloadedConstructor() throws IOException {
-    Connection hbaseConnection =
-        new BigtableConnection(configuration, false, Executors.newSingleThreadExecutor(), null);
+    try (Connection hbaseConnection =
+        new BigtableConnection(configuration, false, Executors.newSingleThreadExecutor(), null)) {
 
-    assertTrue(hbaseConnection.getAdmin() instanceof BigtableAdmin);
-    assertTrue(hbaseConnection.getTable(TABLE_NAME) instanceof AbstractBigtableTable);
+      assertTrue(hbaseConnection.getAdmin() instanceof BigtableAdmin);
+      assertTrue(hbaseConnection.getTable(TABLE_NAME) instanceof AbstractBigtableTable);
```

Comment:
should this be `instanceof BigtableTable`?

Reply:
done